pkgs/desktops/budgie/budgie-control-center/paths.patch

```diff
@@ -0,0 +1,198 @@
+@@ -522,7 +522,7 @@ options_button_clicked_cb (NetDeviceMobile *self)
+ 
+         connection = net_device_get_find_connection (self->client, self->device);
+         uuid = nm_connection_get_uuid (connection);
+-        cmdline = g_strdup_printf ("nm-connection-editor --edit %s", uuid);
++        cmdline = g_strdup_printf ("@networkmanagerapplet@/bin/nm-connection-editor --edit %s", uuid);
+         g_debug ("Launching '%s'\n", cmdline);
+         if (!g_spawn_command_line_async (cmdline, &error))
+                 g_warning ("Failed to launch nm-connection-editor: %s", error->message);
+diff --git a/panels/printers/pp-host.c b/panels/printers/pp-host.c
+index a31a606e3..ed5133d29 100644
+--- a/panels/printers/pp-host.c
++++ b/panels/printers/pp-host.c
+@@ -256,7 +256,7 @@ _pp_host_get_snmp_devices_thread (GTask        *task,
+   devices = g_ptr_array_new_with_free_func (g_object_unref);
+ 
+   argv = g_new0 (gchar *, 3);
+-  argv[0] = g_strdup ("/usr/lib/cups/backend/snmp");
++  argv[0] = g_strdup ("@cups@/lib/cups/backend/snmp");
+   argv[1] = g_strdup (priv->hostname);
+ 
+   /* Use SNMP to get printer's informations */
+diff --git a/panels/user-accounts/cc-user-panel.c b/panels/user-accounts/cc-user-panel.c
+index d70f1efd5..8b86ec655 100644
+--- a/panels/user-accounts/cc-user-panel.c
++++ b/panels/user-accounts/cc-user-panel.c
+@@ -1203,7 +1203,7 @@ spawn_malcontent_control (CcUserPanel *self)
+         /* no-op if the user is administrator */
+         if (act_user_get_account_type (user) != ACT_USER_ACCOUNT_TYPE_ADMINISTRATOR) {
+                 const gchar *argv[] = {
+-                        "malcontent-control",
++                        "@malcontent-ui@/bin/malcontent-control",
+ #ifdef HAVE_MALCONTENT_0_10
+                         "--user",
+                         act_user_get_user_name (user),
+diff --git a/panels/user-accounts/run-passwd.c b/panels/user-accounts/run-passwd.c
+index 86f53d4fc..0b052856f 100644
+--- a/panels/user-accounts/run-passwd.c
++++ b/panels/user-accounts/run-passwd.c
+@@ -150,7 +150,7 @@ spawn_passwd (PasswdHandler *passwd_handler, GError **error)
+         gchar  **envp;
+         gint    my_stdin, my_stdout, my_stderr;
+ 
+-        argv[0] = "/usr/bin/passwd";    /* Is it safe to rely on a hard-coded path? */
++        argv[0] = "/run/wrappers/bin/passwd";    /* Is it safe to rely on a hard-coded path? */
+         argv[1] = NULL;
+ 
+         envp = g_get_environ ();
+diff --git a/panels/user-accounts/user-utils.c b/panels/user-accounts/user-utils.c
+index 0de83479e..628e35247 100644
+--- a/panels/user-accounts/user-utils.c
++++ b/panels/user-accounts/user-utils.c
+@@ -497,7 +497,7 @@ is_valid_username_async (const gchar *username,
+          * future, so it would be nice to have some official way for this
+          * instead of relying on the current "--login" implementation.
+          */
+-        argv[0] = "/usr/sbin/usermod";
++        argv[0] = "@shadow@/bin/usermod";
```

Comment:
I looked up every reference to `/usr` and found this. This isn't in GNOME Control Center's `paths.patch`, I'm not sure if that's on purpose?

Reply:
This is previously patched in https://github.com/NixOS/nixpkgs/commit/1de94d59af634516ce1085f6e99f2a33b086e22f but it's later dropped in https://github.com/NixOS/nixpkgs/commit/fcf2213f74557a1a6c6d3ce677caefae9b7f5cfa, not sure the reason. Is adding users in `*-control-center` actually works with e.g. `users.mutableUsers` enabled?

(Ha! Saw @jtojnar in [gnome-control-center!653](https://gitlab.gnome.org/GNOME/gnome-control-center/-/merge_requests/653#note_678656))

Reply:
Hmm, maybe it was removed accidentally, though it does not look like the surrounding code changed to cause a merge conflict.

Reply:
> Is adding users in *-control-center actually works with e.g. users.mutableUsers enabled

I was able to create a new user and log in with its account.

<details><summary>Screenshots</summary>

![Captura desde 2023-03-29 00-51-47](https://user-images.githubusercontent.com/62166915/228422379-2ded9ce1-08c0-4568-b4fb-5255cda2be81.png)
![imagen](https://user-images.githubusercontent.com/62166915/228422686-5a3b5f44-bf0e-49a3-8bb7-8dce9a660407.png)

</details>